src/main/java/com/fundation/logic/view/AdvancedSearchPanel.java

```diff
@@ -0,0 +1,158 @@
+        gbc_label.insets = new Insets(0, 0, 5, 5);
+        gbc_label.gridx = 0;
+        gbc_label.gridy = 4;
+        add(label, gbc_label);
+
+        JSeparator separator = new JSeparator();
+        GridBagConstraints gbc_separator = new GridBagConstraints();
+        gbc_separator.insets = new Insets(0, 0, 5, 5);
+        gbc_separator.gridx = 0;
+        gbc_separator.gridy = 5;
+        add(separator, gbc_separator);
+
+        JLabel lblAudio = new JLabel("Audio");
+        lblAudio.setFont(new Font("Tahoma", Font.BOLD, 14));
+        GridBagConstraints gbc_lblAudio = new GridBagConstraints();
+        gbc_lblAudio.insets = new Insets(0, 0, 5, 0);
+        gbc_lblAudio.gridwidth = 2;
+        gbc_lblAudio.gridx = 0;
+        gbc_lblAudio.gridy = 6;
+        add(lblAudio, gbc_lblAudio);
+
+        JLabel lblCodecAudio = new JLabel("Codec: ");
+        GridBagConstraints gbc_lblCodecAudio = new GridBagConstraints();
+        gbc_lblCodecAudio.anchor = GridBagConstraints.EAST;
+        gbc_lblCodecAudio.insets = new Insets(0, 0, 5, 5);
+        gbc_lblCodecAudio.gridx = 0;
+        gbc_lblCodecAudio.gridy = 7;
+        add(lblCodecAudio, gbc_lblCodecAudio);
+
+        JComboBox comboBoxAudioCodec = new JComboBox();
+        GridBagConstraints gbc_comboBoxAudioCodec = new GridBagConstraints();
+        gbc_comboBoxAudioCodec.insets = new Insets(0, 0, 5, 0);
+        gbc_comboBoxAudioCodec.fill = GridBagConstraints.HORIZONTAL;
+        gbc_comboBoxAudioCodec.gridx = 1;
+        gbc_comboBoxAudioCodec.gridy = 7;
+        add(comboBoxAudioCodec, gbc_comboBoxAudioCodec);
+
+        JLabel lblChannels = new JLabel("Channels: ");
+        GridBagConstraints gbc_lblChannels = new GridBagConstraints();
+        gbc_lblChannels.anchor = GridBagConstraints.EAST;
+        gbc_lblChannels.insets = new Insets(0, 0, 5, 5);
+        gbc_lblChannels.gridx = 0;
+        gbc_lblChannels.gridy = 8;
+        add(lblChannels, gbc_lblChannels);
+
+        JComboBox comboBoxAudioChannels = new JComboBox();
+        GridBagConstraints gbc_comboBoxAudioChannels = new GridBagConstraints();
+        gbc_comboBoxAudioChannels.insets = new Insets(0, 0, 5, 0);
+        gbc_comboBoxAudioChannels.fill = GridBagConstraints.HORIZONTAL;
+        gbc_comboBoxAudioChannels.gridx = 1;
+        gbc_comboBoxAudioChannels.gridy = 8;
+        add(comboBoxAudioChannels, gbc_comboBoxAudioChannels);
+
+        JLabel lblBitrate = new JLabel("Bitrate: ");
+        GridBagConstraints gbc_lblBitrate = new GridBagConstraints();
+        gbc_lblBitrate.anchor = GridBagConstraints.EAST;
+        gbc_lblBitrate.insets = new Insets(0, 0, 5, 5);
+        gbc_lblBitrate.gridx = 0;
+        gbc_lblBitrate.gridy = 9;
+        add(lblBitrate, gbc_lblBitrate);
+
+        textFieldAudioBitrate = new JTextField();
+        GridBagConstraints gbc_textFieldAudioBitrate = new GridBagConstraints();
+        gbc_textFieldAudioBitrate.insets = new Insets(0, 0, 5, 0);
```

Comment:
Magical numbers.

Reply:
Numbers of coordinates in GridBagLayout, please refactor in the next Sprint